tests/models/test_restore.py

```diff
@@ -309,7 +309,7 @@ def test_model_saving_loading(tmpdir):
     hparams_path = os.path.join(hparams_path, 'hparams.yaml')
     model_2 = EvalModelTemplate.load_from_checkpoint(
         checkpoint_path=new_weights_path,
-        hparams_file=hparams_path
+        hparams_file=hparams_path,
```

Comment:
is this a good practice thing, adding a comma even after the last param?

Reply:
I guess black does it too as with adding new parameter dif shows changed just one line instead of two if you have to add "," to the previous one...